client/src/components/Proposals/ProposalsTimelineWidget/ProposalsTimelineWidget.tsx

```diff
@@ -1,14 +1,17 @@
 import { isAfter, isSameDay, isWithinInterval } from 'date-fns';
 import { motion, useMotionValue } from 'framer-motion';
-import React, { ReactElement, useLayoutEffect, useRef } from 'react';
+import React, { ReactElement, useCallback, useEffect, useRef } from 'react';
 
 import ProposalsTimelineWidgetItem from 'components/Proposals/ProposalsTimelineWidgetItem';
 import getMilestones, { Milestone } from 'constants/milestones';
 
 import styles from './ProposalsTimelineWidget.module.scss';
 
+let isInitialResizeDone = false;
```

Comment:
`useState` would be more react-way. Causes unintentional rerenders?

Reply:
This `let` is used intentionally. We ignore first resize only once (not every time when user opens ProjectsView).